addon/macros/has-many-through.js

```diff
@@ -22,6 +22,11 @@ export default function (...args) {
       if (!self.isDestroyed) {
         self.notifyPropertyChange(key);
       }
+    },
+    observerFunction2 = function () {
```

Comment:
Maybe observerFunctionForRejected is a better name?

Reply:
Aaaaah! I was trying to work out what the observer functions were for. Yes—clearer naming would definitely help 🙂

Reply:
@omersaleem done thanks